test/K6/src/api/platform/authorization/maskinporten.js

```diff
@@ -78,6 +78,25 @@ export function postMaskinportenSchema(altinnToken, offeredByPartyId, resourceid
   return http.post(endpoint, bodystring, params);
 }
 
+/**
+ * GET call to get maskinportenschemas that have been received by the current party
+ * @param {*} altinnToken personal token for DAGL
+ * @param {*} offeredByOrganizationNumber the organization number for the offeredby party
+ * @param {*} resourceid the id of the resource to delegate
+ * @param {*} attributeId the attribute id for the receiver of the schema. 'urn:altinn:partyid' or 'urn:altinn:organizationnumber'
```

Comment:
attributeId and attributeValue could be better named to represent the party the delegation is made to

Reply:
renamed them to toAttributeId/toAttributeValue fromAttributeId/FromAttributeValue